components/x-privacy-manager/src/__tests__/helpers.js

```diff
@@ -0,0 +1,51 @@
+export const CONSENT_PROXY_HOST = 'https://consent.ft.com'
+export const CONSENT_PROXY_ENDPOINT = 'https://consent.ft.com/__consent/consent-record/FTPINK/abcde'
+
+export const buildPayload = (consent) => ({
+	consentSource: 'consuming-app',
+	data: {
+		behaviouralAds: {
+			onsite: {
+				fow: 'privacyCCPA/H0IeyQBalorD.6nTqqzhNTKECSgOPJCG',
+				lbi: true,
```

Comment:
what are FoW and LBI?

Reply:
`fow`: Form of Words
`lbi`: Legitimate Business Interest

They're parameters required by the Single Consent Store

Reply:
nice, thanks :)